src/components/ArticleCard.jsx

```diff
@@ -0,0 +1,17 @@
+import ReactTimeAgo from 'react-time-ago';
+
+export const ArticleCard = ({
+  article: { topic, created_at, title, article_img_url, votes, comment_count },
+}) => {
+  return (
+    <li className='article-card'>
+      <span>{topic}, </span>
+      <ReactTimeAgo date={new Date(created_at).getTime()} locale='en-US' />
```

Comment:
This is a nice find! I've used day.js before but will have to look at how this works. Always better than a UTC string.